### PR TITLE
Adding missing unit tests

### PR DESCRIPTION
**Pull request by elabeca**

### CODE REVIEW

File: app/controllers/EnquiryController.scala [hunk 1]
```diff
@@ -53,8 +52,6 @@ class EnquiryController @Inject()(appConfig: AppConfig,
     implicit request =>
       authorised(Enrolment("HMRC-NI")) {
         Future.successful(Ok(enquiry(appConfig, form, options)))
-      } recoverWith {
-        case InsufficientEnrolments(msg) => Future.successful(Unauthorized(msg))
```
Comment (sammhmrc) [inline app/controllers/EnquiryController.scala]: Was this intensional?

File: test/connectors/TwoWayMessageConnectorSpec.scala [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        }
+      })
+      .build()
+  }
+
+  val twoWayMessageConnector = injector.instanceOf[TwoWayMessageConnector]
+
+  "postMessage" should {
+
+    "respond with a mongo id after a successful call to two-way-message service results in a message creation from a valid payload" in {
```
Comment (sammhmrc) [inline test/connectors/TwoWayMessageConnectorSpec.scala]: Can we add a test for a failing case? Make sure that responds correctly.

File: app/controllers/EnquiryController.scala (hunk 1 above)
Comment (elabeca) [reply app/controllers/EnquiryController.scala]: Yes, the handling of auth related errors should be handled by the class ErrorHandler. That can only be tested via integration tests hence the lack of unit tests around that. There's a reason for that but unfortunately I don't remember why, but speaking to Peter Anning should explain that.

File: test/connectors/TwoWayMessageConnectorSpec.scala (hunk 2 above)
Comment (elabeca) [reply test/connectors/TwoWayMessageConnectorSpec.scala]: Done

**Comment by andy1138** [comment]
👍

**Pull request merged**